reana-ui/src/pages/launchOnReana/LaunchOnReana.js

```diff
@@ -0,0 +1,75 @@
+/*
+  This file is part of REANA.
+  Copyright (C) 2022 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { Button, Container, Loader } from "semantic-ui-react";
+
+import BasePage from "../BasePage";
+import { Title } from "~/components";
+import { errorActionCreator } from "~/actions";
+import client from "~/client";
+import { useQuery } from "~/hooks";
+import { LAUNCH_ON_REANA_PARAMS_WHITELIST } from "~/config";
+
+import styles from "./LaunchOnReana.module.scss";
+
+export default function LaunchOnReana() {
+  const [loading, setLoading] = useState(false);
+  const history = useHistory();
+  const query = useQuery();
+  filterParams(query);
+  const dispatch = useDispatch();
+
+  function handleLaunch() {
+    setLoading(true);
+    client
+      .launchWorkflow(Object.fromEntries(query))
+      .then(({ data: { workflow_id: workflowId } }) => {
+        history.push(`/details/${workflowId}`);
+      })
+      .catch((err) => {
+        dispatch(errorActionCreator(err));
+      })
+      .finally(() => setLoading(false));
+  }
+
+  function filterParams(query) {
+    for (var qsParamsKey of query.keys()) {
```

Comment:
Minor: could we use `let` here instead of `var`?

Also, if we would use `queryString.parse()` instead of `URLSearchParams()` inside `useQuery()` hook, we could use `filter` and `reduce` here to filter our the list.. perhaps worth switching?

Reply:
> Minor: could we use `let` here instead of `var`?

Got tricked by the [MDN example](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/keys#examples) 😅 

I think `const` would be enough.